src/acl.c

```diff
@@ -836,7 +854,13 @@ sds ACLDescribeUser(user *u) {
         }
         sdsfree(default_perm);
     }
-    return res;
+
+    if (u->acl_string) decrRefCount(u->acl_string);
```

Comment:
no need this check again, `u->acl_string` is NULL for sure, since it was validated at the beginning of this method.